src/additions/test_test_pass.py

```diff
@@ -0,0 +1,40 @@
+"""PASS statement
+
+@see: https://docs.python.org/3/tutorial/controlflow.html
+
+The pass statement does nothing. It can be used when a statement is required syntactically but
+the program requires no action.
+"""
+
+def test_pass_function():
+    pass
+
+def test_pass_in_loop():
+    """PASS in loops.
+
+    "Pass" can be used when a statement is required syntactically but the program requires no
+    action. For example:
+    """
+
+    # pylint: disable=unused-variable
```

Comment:
Can we avoid disabling this pylint warnings?

Reply:
we can do this by either not declaring any unwanted variable or to make sure that we use all variable. in this test example we are showing the use of pass. we are making classes and declaring function definitions while instead of the body, we are only using pass. in real cases, these pass will be replaced by a function body and the warning could be avoided